components/dot_menu/dot_menu.tsx

```diff
@@ -160,7 +181,7 @@ export default class DotMenu extends React.PureComponent {
     }
 
     // listen to clicks/taps on add reaction menu item and pass to parent handler
-    handleAddReactionMenuItemActivated = (e) => {
+    handleAddReactionMenuItemActivated = (e: Event) => {
```

Comment:
I believe the correct type for this is
```suggestion
    handleAddReactionMenuItemActivated = (e: React.MouseEvent) => {
```